Sources/Comet/CometClient/CometClient.swift

```diff
@@ -16,23 +16,31 @@ public final class CometClient {
     private let authenticator: Authenticator
     private let authenticatedRequestBuilder: AuthenticatedRequestBuilding
     private let requestResponseHandler: RequestResponseHandling
+    private let logLevel: RequestLogLevel
+    private let logger: (String) -> Void
 
     /// TODO
     /// - Parameters:
     ///   - urlSession: TODO
     ///   - tokenProvider: TODO
     ///   - authenticatedRequestBuilder: TODO
     ///   - requestResponseHandler: TODO
+    ///   - logLevel: TODO
```

Comment:
Please describe the parameters in this init method.

Reply:
Described ✅